modules/porous_flow/doc/content/modules/porous_flow/sqa/porous_flow_srs.md

```diff
@@ -1 +1,33 @@
 !template load file=sqa/module_srs.md.template category=porous_flow module=Porous Flow
+
+!template! item key=system-scope
+!! system-scope-begin
+The Porous Flow module is a library of physics for fluid and heat flow in porous media. It is formulated
+in an extremely general manner, so is capable of solving problems with an arbitrary number of phases
+(gas, liquid, etc) and fluid components (species present in each fluid phase), using any set of primary
+variables.
+
+By simply adding pieces of physics together in an input file, the Porous Flow module enables the user
+to model problems with any combination of fluid, heat and geomechanics.
+!! system-scope-finish
+!template-end!
+
+!template! item key=system-purpose
+!! system-purpose-begin
+The purpose of the [!ac](MOOSE) Porous Flow module is to provide functionality to support simulations
+for fluid and heat flow in porous media. The Porous Flow module can be easily coupled with other physics
+modules in [!ac](MOOSE).
```

Comment:
The intent of the system purpose section is to describe where this module might be used and potentially how. I gave a brief intro here, but more content is needed.

Reply:
Some typical usage examples include groundwater flow, geothermal modelling, geological storage of CO2, and long-term mineralization due to geochemical reactions.

Reply:
Added this line to the current version.